### PR TITLE
Gh pages

### PR DESCRIPTION
**Pull request by rkelson**
more dialing in pages

### CODE REVIEW

File: data/sgid-cadastre/parcels/index.html [hunk 1]
```diff
@@ -123,49 +114,52 @@ <h5 id="UtahParcels-updates">Updates</h5>
...
-August 2012 - Utah County<br />
-July 2012 - Salt Lake County<br />
-June 2012 - Salt Lake County, Kane County, Washington County, Utah County, Weber County, Cache County<br />
-March 2012<br />
-August 2011<br />
-December 2010<br />
-July 2010<br />
+<ul>
+<p>
+    <li>March 2016 - Utah County, Salt Lake County, Washington County, Carbon County<br /></li>
```
Comment (steveoh) [inline data/sgid-cadastre/parcels/index.html]: remove all the br's that are within the li's. They are extra and unnecessary.

**Comment by steveoh** [comment]
thanks.

**Comment by rkelson** [comment]
no, thank you

Rick Kelson
Utah AGRC
801.538.3237
gis.utah.gov

**Comment by steveoh** [comment]
no thank him.

**Pull request closed without merging**